app/javascript/components/layout/CustomTable.js

```diff
@@ -347,7 +347,7 @@ CustomTable.propTypes = {
   entryOptions: PropTypes.array,
   getRowClassName: PropTypes.func,
   getCustomTableClassName: PropTypes.func,
-  currentUser: PropTypes.string,
+  getAriaLabelText: PropTypes.func,
```

Comment:
It's not super clear to a user of `CustomTable` what this aria label text is for by the name, so I'd just suggest renaming to something like `getTableRowAriaLabel`.

Reply:
good catch, updated here: https://github.com/SaraAlert/SaraAlert/pull/768/commits/6e1b07436033b48d821ebdd2126ce88765f3e94b